timbermill-java/timbermill-server/src/test/java/com/datorama/oss/timbermill/TimberLogAdvancedTest.java

```diff
@@ -99,6 +99,26 @@ public void testOngoingTask() {
         assertEquals(string3, ongoingTask.getString().get(string3));
     }
 
+    public void testTaskReachedRateLimit() {
+        String ongoingTaskName = EVENT + "_rate_limit";
+        String limitStr = System.getenv("LIMIT_FOR_PERIOD");
```

Comment:
Don't see the point in that, just set a value

Reply:
we spoke about this